doc/contributing/releases.md

```diff
@@ -603,13 +603,14 @@ the build before moving forward. Use the following list as a baseline:
   must be in the expected updated version)
 * npm version (check it matches what we expect)
 * Run the test suite against the built binaries (optional)
+  * Remember to use the proposal branch
+  * Run `make build-addons` before running the tests
+  * Remove `config.gypi` file
 
 ```console
 ./tools/test.py --shell ~/Downloads/node-v18.5.0-linux-x64/bin/node
 ```
 
-<sup>There may be test issues if the branch used to test does not match the Node.js binary.</sup>
```

Comment:
That's still true, if you are running tests meant for Node.js 20.x-pre with a 14.x binary, I would expect that all tests that concern semver-major changes would fail. But maybe that's obvious and doesn't need to be in this document, in which case I'm fine with removing it.